### PR TITLE
Added support to switch branch in basic mode

### PR DESCRIPTION
**Pull request by isc-akadoo**

### CODE REVIEW

**Comment by isc-tleavitt** [comment]
@isc-akadoo if you rebase/merge from main the CI issues should be resolved.

**Comment by isc-tleavitt** [comment]
@isc-akadoo nevermind, I just did it.

File: CHANGELOG.md [hunk 1]
```diff
@@ -163,3 +163,5 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [2.0.1] - 2022-06-02
 - Last released version before CHANGELOG existed.
 
+### Fixed
```
Comment (isc-tleavitt) [inline CHANGELOG.md:166]: @isc-akadoo this belongs under "Fixed" the 2.5.0 section for the upcoming release.

**Review by isc-tleavitt — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (isc-akadoo) [reply CHANGELOG.md]: Moved to the upcoming release section.

**Review by isc-tleavitt — APPROVED** [review]

**Pull request merged**